ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -103,16 +103,23 @@ interface {{ port_channel_interface }}
 {%     if port_channel_interfaces[port_channel_interface].qos.cos is arista.avd.defined %}
    qos cos {{ port_channel_interfaces[port_channel_interface].qos.cos }}
 {%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].shape.rate is arista.avd.defined %}
```

Comment:
what was the reason to move shape.rate as part of this change?

Reply:
This was moved to match the cli output order. This was requested in this PR.